wannierberri/__sym_wann.py

```diff
@@ -0,0 +1,625 @@
+import numpy as np
+import spglib
+import numpy.linalg as la
+import sympy as sym
+
+#np.set_printoptions(threshold=np.inf,linewidth=500)
+np.set_printoptions(suppress=True,precision=4,threshold=np.inf,linewidth=500)
+
+
+class sym_wann():
```

Comment:
Class names should normally use the CapWords convention. So `SymWann` would be better.
https://peps.python.org/pep-0008/#class-names

Reply:
Done